spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -370,6 +370,27 @@ public class ConverterConfiguration {
 }
 ----
 
+==== Structs in Columns
```

Comment:
This section best follows the custom converter section, because its behavior is very heavily dependent on knowing about customer converters. The behavior of reading of STRUCTs is entirely dependent on the inclusion of a custom converter. In fact I considered not having a header at all. 

I also think the very small example is better than extending the running example of Trade and Trader. That is a great example to show the other features of this project that require more context, but this specific feature can be demonstrated in isolation. The reason is there is a lot of irrelevant code and context in the example you included. It took some effort to focus on what feature is specifically being demonstrated. I think the smaller example presents the relevant features in a easier-to-digest format.

Reply:
Why is it heavily dependent on custom converters? I just tried the feature out without having to define any converters.

Reply:
The goal of this PR is to make this line true: 
"If a `Converter<Struct, A>` is provided, then properties of type `List<A>` can be used."

Reply:
I guess there are two things going on here. Support for structs, and customer converters for structs. Perhaps, the documentation should be separated into two sections then: "Arrays of Structs" and "Customer converters for Structs". Does it make sense?

Reply:
Yes that sounds good. I'll move things around.

Reply:
done.